src/schemas/user.schema.ts

```diff
@@ -100,3 +100,24 @@ export const patchUserSchema = z.object({
  * @author aayushchugh
  */
 export type PatchUserSchema = z.infer<typeof patchUserSchema>;
+
+/**
+ * This schema is used to validate `POST /users/:id/marketing-emails/unsubscribe` request
+ *
+ * @author aayushchugh
+ */
+export const unsubscribeUserFromMarketingEmailSchema = z.object({
```

Comment:
```suggestion
export const getUnsubscribeUserFromMarketingEmailSchema = z.object({
```

Not sure if we should do this, just to stay consistent